routes/articleRoutes.js

```diff
@@ -0,0 +1,19 @@
+import express from 'express';
+import Articles from '../controllers/article';
+import { verifyToken, articleValidation } from '../middlewares';
+
+const router = express.Router();
+
+const { createArticle } = Articles;
+const {
+  expectedParamsValidator,
+  nonEmptyParamsValidator
+} = articleValidation;
+
+router.route('/articles')
+  .all(verifyToken)
+  .post(expectedParamsValidator,
```

Comment:
`expectedParamsValidator` should be on the next line.

Reply:
Okay, I'll do so